src/main/java/br/unb/cic/analysis/AbstractMergeConflictDefinition.java

```diff
@@ -87,8 +91,11 @@ public void loadSinkStatements() {
                 if(body == null) continue;
                 for(Unit u: body.getUnits()) {
                     if(definitions.get(className).contains(u.getJavaSourceStartLineNumber())) {
-                        Statement stmt = createStatement(m, u, type);
+                        Statement stmt = createStatement(m, u, type); 
                         statements.add(stmt);
+                        if (stmt.isSource()) {
```

Comment:
Por que só adicionar se for source?
Se existir um cenário que sink adiciona e source remove, não vai adicionar?